stob/domain.c

```diff
@@ -171,7 +171,7 @@ static int stob_domain_init_create(const char *location,
 				       "destroy() failed: rc = %d, "
 				       "location = %s", rc1, location);
 			}
-			/* rc1 is lost here */
+			/* rc1 is lost here */ 
```

Comment:
No trailing space at the end of a line.

Reply:
removed, actually it is accidentally inserted while browsing